rabbitmq/plan.sh

```diff
@@ -29,7 +29,7 @@ pkg_build_deps=(
   core/rsync
   core/unzip
   core/zip
-  core/elixir
+  core/elixir/1.6.5
```

Comment:
Rather than pin,  could we roll the version of rabbitmq forward?  Policy is to not pin, as per: https://github.com/habitat-sh/core-plans/blob/master/CONTRIBUTING.md#dependencies   

If rolling forward doesn't fix and we need to pin,  would you mind opening an issue to remove the pin when this is resolved upstream?